invenio_app_ils/ui/src/invenio_app_ils/common/api/loan.js

```diff
@@ -44,6 +44,20 @@ const buildPendingQuery = (documentPid, itemPid) => {
   return `${qsDocItem} AND state:PENDING`;
 };
 
+const assignLoanItem = (loanPid, itemPid) => {
+  const newItemRef = {
+    $ref: `https://127.0.0.1:5000/api/resolver/circulation/items/${itemPid}`,
```

Comment:
I think we should re-think how we do this... we cannot hardcode this ref here.
Probably, instead of doing a patch, we need to create a new API endpoint, let's say `/loan/changeItem`, passing the `itemPid`, and server side it performs a patch.
WDYT?